<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Изменение скорости</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="velocityXLabel">
       <property name="text">
        <string>Скорость X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="velocityXLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="velocityYLabel">
       <property name="text">
        <string>Скорость Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="velocityYLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="velocityZLabel">
       <property name="text">
        <string>Скорость Z</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="velocityZLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="durationLabel">
       <property name="text">
        <string>Длительность</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="durationLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="confirm_button">
     <property name="text">
      <string>Выполнить изменения скорости</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
